spellbook/models/seaport/ethereum/seaport_ethereum_transfers.sql

```diff
@@ -0,0 +1,822 @@
+                                              {% endif %}
+          left join nft_ethereum.aggregators agg ON agg.contract_address = tx.to
+          left join tokens_ethereum.nft n on n.contract_address = nft_contract_address
+                    left join tokens_ethereum.erc20 t1 on t1.contract_address = 
+          case when a.original_currency_contract = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+               else a.original_currency_contract end                                
+          left join prices.usd p1 on p1.contract_address = 
+          case when a.original_currency_contract = '0x0000000000000000000000000000000000000000' then 
+          '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+               else a.original_currency_contract end
+          and p1.minute = date_trunc('minute', a.block_time)
+          and p1.blockchain = 'ethereum')
+          
+,p2_call as (
+    select 'available_advanced_orders' as main_type
+          ,'bulk' as sub_type
+          ,idx as sub_idx
+          ,get_json_object(get_json_object(each, "$.parameters"), "$.zone") as zone
+          ,get_json_object(get_json_object(each, "$.parameters"), "$.offerer") as offerer
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.offer[0]"), "$.token") as offer_token
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.offer[0]"), "$.itemType") as offer_item_type
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.offer[0]"), "$.identifierOrCriteria") as offer_identifier
+          ,get_json_object(get_json_object(each, "$.parameters"), "$.orderType") as offer_order_type
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.consideration[0]"), "$.token") as price_token
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.consideration[0]"), "$.itemType") as price_item_type
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.consideration[0]"), "$.startAmount") as price_amount
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.consideration[1]"), "$.startAmount") as fee_amount
+          ,get_json_object(get_json_object(get_json_object(each, "$.parameters"), "$.consideration[2]"), "$.startAmount") as royalty_amount
+          ,c.call_tx_hash as tx_hash
+          ,c.call_block_time as block_time
+          ,c.call_block_number as block_number
+          ,c.contract_address as exchange_contract_address
+      from (select *, posexplode(advancedOrders) as (idx, each) from {{ source('seaport_ethereum','Seaport_call_fulfillAvailableAdvancedOrders') }}) c
+     where call_success
+      {% if is_incremental() %} -- this filter will only be applied on an incremental run 
```

Comment:
this one is fine compared to p1_call, but if you wanted, you could also move within the () subquery on line 245